openqaoa/problems/minimumvertexcover.py

```diff
@@ -0,0 +1,178 @@
+#   Copyright 2022 Entropica Labs
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+import networkx as nx
+
+from .helper_functions import check_kwargs
+from .problem import Problem
+from .qubo import QUBO
+
+from collections import Counter
```

Comment:
From PEP8 (see https://peps.python.org/pep-0008/#imports)

> Imports are always put at the top of the file, just after any module comments and docstrings, and before module globals and constants.
> Imports should be grouped in the following order:
>  - Standard library imports.
>  - Related third party imports.
>  - Local application/library specific imports.

Let's start by making sure this folder follows the right convention. (Worth checking it black does it automatically)

Reply:
black doesnt do this automatically. I'll fix this.